lib/oli_web/live/sections/assessment_settings/settings_live.ex

```diff
@@ -254,9 +254,11 @@ defmodule OliWeb.Sections.AssessmentSettings.SettingsLive do
 
   defp get_assessments(section_slug, student_exceptions) do
     DeliveryResolver.graded_pages_revisions_and_section_resources(section_slug)
-    |> Enum.map(fn {rev, sr} ->
+    |> Enum.with_index()
+    |> Enum.map(fn {{rev, sr}, index} ->
       Settings.combine(rev, sr, nil)
       |> Map.merge(%{
+        index: index + 1,
```

Comment:
NIT: [Enum.with_index](https://hexdocs.pm/elixir/Enum.html#with_index/2) allows us to provide an offset as a second argument, so we could just do `|> Enum.with_index(1)` and then just do `index: index` instead of `index: index + 1`